dockerfiles/geant-ca

```diff
@@ -0,0 +1,39 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I would not place this here, this does not seem like a good place for us to have something like this in here. we can maybe use openssl commands to retrieve it or keep track of it, especially if it for development.

It can be mounted with a volume configuration for either test or production.

Reply:
I agree. I was wondering what would be the best way to solve this issue. We don't use that Dockerfile in production, but only in testing or local dev.

Then there should be a script as `Entrypoint`, which looks for a certificate, and if it's found, it's appended.

Or is it a good solution to pass the certificate as build `ARG`?

Reply:
this does not seem like a build `ARG` to justify that, it seems like something we can mount with a volume or use with a normal environment variable.

Reply:
I don't see how to append the certificate to the existing file with your suggestion. Do you have an example?

Reply:
Removed the cert

Reply:
example, the `requests.get` can be a path or a volume, and it can be done in a different stage of the docket build
```
customca = requests.get('http://place-to-download-your-root-ca-pem.file').content 
cafile = certifi.where() 
with open(cafile, 'ab') as outfile: 
    outfile.write(b'\n')
    outfile.write(customca)
```

Reply:
you can also use the same method you used with `cat` and `>>`, however you load the certificate from a build arg as you initially said, i am not sure i have a solution with env var :thinking:

Reply:
Your example above is what I had in mind for env var.

Reply:
The cert is not necessary, after all. There was a misconfiguration in the OIDC provider that the intermediary cert was not provided.